babel.config.js

```diff
@@ -6,7 +6,7 @@ module.exports = (api) => {
       '@babel/preset-env',
       {
         useBuiltIns: 'usage',
-        corejs: 3,
+        corejs: '3.6.5',
```

Comment:
Did the spec change so this differs now? Why do you point an exact version? If we upgrade automatically, dependabot will not track this line. I would rather have only 3, it will indicate the most recent but I am not sure.

Reply:
it was not picking the polyfill for allSettled which was added in 3.2.x. I guess it has something to do with this https://github.com/zloirock/core-js#babelpreset-env